pkg/ibmvpc/examples/out/explain_out/LBToResIPNode_all_vpcs_explain_debug.txt

```diff
@@ -35,7 +35,7 @@ Path is disabled; The relevant rules are:
 ------------------------------------------------------------------------------------------------------------------------
 
 No connections from from kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca[LB private IP][192.168.40.6] to iks-clusterid:1[192.168.32.5];
-connection is blocked by Load Balancer
+connection can not be initiated by Load Balancer
```

Comment:
"can not be initiated " => "will not be initiated"

Reply:
fixed